cypress.config.js

```diff
@@ -67,7 +66,8 @@ module.exports = defineConfig({
 			if (semver.satisfies(config.env.phpSemverVersion, '<7.3.0')) {
 				config.excludeSpecPattern = config.excludeSpecPattern.concat(
 					[
-						"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/steps/ecommerce/**",
+						"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/2-ecommerce/**",
+						"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/1-top-priority/TC_07-top-priority.cy.js"
```

Comment:
Why are we skipping top-priority for PHP < `7.3.0`? We can skip the `selling` card click condition in the test itself?

Reply:
Then we need to specify for php version on top-priority page for not selecting selling.